README.md

```diff
@@ -94,6 +94,18 @@ $ cat echo.cwl | docker run --rm -i ttanjo/cwl-inspector - .cwlVersion
 --- v1.0
 ```
 
+## using job parameter file
+
+```console
+$ cwltool --make-template echo.cwl > echo.yml
+$ cat echo.yml
+input: a_string  # type "string" (optional)
+$ cat echo.cwl | docker run --rm -i ttanjo/cwl-inspector - echo.yml commandline
```

Comment:
This example does not work because:

- `echo.yml` is not located in the container.
- Need `-i` before `echo.yml`
- `echo.cwl` need `DockerRequirement` as a requirement. I will fix it later.

Please fix the command as follows:
```console
$ cat echo.cwl | docker run --rm -i -v $PWD/echo.yml:/workdir/echo.yml --workdir=/workdir ttanjo/cwl-inspector:v0.0.6 -i echo.yml - commandline
env HOME='/workdir' TMPDIR='/tmp' /bin/sh -c 'cd ~ && "cowsay" "a_string"' > /workdir/output
```